src/lib.rs

```diff
@@ -308,7 +308,7 @@ impl Iterator for AllBytesIter {
     type Item = u8;
     #[inline]
     fn next(&mut self) -> Option<Self::Item> {
-        if self.0 <= 256 {
+        if self.0 < 256 {
```

Comment:
Oops 😓  It does not appear to have caused any major issues though since I think all uses of this are idempotent.

Reply:
Ah no worries! Thanks for the regression test. :-)